### PR TITLE
circleci: switch to Quay

### PR DESCRIPTION
**Pull request by fabxc**
@sdurrheimer

### CODE REVIEW

**Comment by sdurrheimer** [comment]
You will need to change images names in the follwing files:
- `README.md`: all mentions to `prom/golang-builder` images
- `Makefile`: `REPOSITORY` var
- `1.5/Makefile.COMMON`: `REPOSITORY` var
- `1.5/arm/Dockerfile`: the `FROM` image
- `1.5/main/Dockerfile`: the `FROM` image
- `1.5/powerpc/Dockerfile`: the `FROM` image
- `1.6/Makefile.COMMON`: `REPOSITORY` var
- `1.6/arm/Dockerfile`: the `FROM` image
- `1.6/main/Dockerfile`: the `FROM` image
- `1.6/powerpc/Dockerfile`: the `FROM` image
- `1.6/mips/Dockerfile`: the `FROM` image

**Comment by fabxc** [comment]
@sdurrheimer does that seem about right.
I liked the shields – maybe the will come for Quay some day.

File: 1.5/Makefile.COMMON [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REPOSITORY    := prom
+REPOSITORY    := prometheus
```
Comment (sdurrheimer) [inline 1.5/Makefile.COMMON]: you need to add `quay.io/` here.

File: 1.6/Makefile.COMMON [hunk 2]
```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REPOSITORY    := prom
+REPOSITORY    := prometheus
```
Comment (sdurrheimer) [inline 1.6/Makefile.COMMON]: same `quay.io/` here.

File: Makefile [hunk 3]
```diff
@@ -11,7 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-REPOSITORY := prom
+REPOSITORY := prometheus
```
Comment (sdurrheimer) [inline Makefile]: same `quay.io/` here.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,6 @@
 # Prometheus Golang builder Docker images
 
 [![CircleCI](https://circleci.com/gh/prometheus/golang-builder/tree/master.svg?style=shield)][circleci]
-[![Docker Stars](https://img.shields.io/docker/stars/prom/golang-builder.svg)][hub]
-[![Docker Pulls](https://img.shields.io/docker/pulls/prom/golang-builder.svg)][hub]
-[![Image Size](https://img.shields.io/imagelayers/image-size/prom/golang-builder/latest.svg)][imagelayers]
-[![Image Layers](https://img.shields.io/imagelayers/layers/prom/golang-builder/latest.svg)][imagelayers]
```
Comment (sdurrheimer) [inline README.md]: maybe

```
[![Docker Repository on Quay.io](https://quay.io/repository/prometheus/golang-builder/status)][quayio]
...
[quayio]: https://quay.io/repository/prometheus/golang-builder
```

**Comment by fabxc** [comment]
done

**Comment by sdurrheimer** [comment]
LGTM :+1: (build failing because of branch name)

**Pull request merged**